test/e2e/density.go

```diff
@@ -155,6 +155,12 @@ var _ = Describe("Density", func() {
 			// Tune the threshold for allowed failures.
 			badEvents := BadEvents(events)
 			Expect(badEvents).NotTo(BeNumerically(">", int(math.Floor(0.01*float64(totalPods)))))
+
+			// Verify latency metrics
+			// TODO: update threshold to 1s once we reach this goal
+			highLatencyRequests, err := HighLatencyRequests(c, 10*time.Second)
```

Comment:
Currently it's impossible to reset those metrics. The only way would be to restart apiserver.

If we really want to count this properly we'd need to read the full histogram (not the summary) at the beginning, and subtract it from the one read at the end. Writing such code is of course doable, but not that straight forward.

My hope is that once we have prometheus server set up we will be able to write more advanced queries and read it easily.

I think that currently the effect you described will not be that huge, because other density tests are much smaller than the biggest one and as a result should not affect overall metrics too much.

If you don't mind I'd submit this change as is (it's better than nothing) and improve this in time.